cgmes/cgmes-shortcircuit/src/test/java/com/powsybl/cgmes/shortcircuit/ShortCircuitCgmesConformity1ModifiedCatalog.java

```diff
@@ -0,0 +1,43 @@
+package com.powsybl.cgmes.shortcircuit; /**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import com.powsybl.cgmes.model.test.TestGridModelResources;
+import com.powsybl.commons.datasource.ResourceSet;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class ShortCircuitCgmesConformity1ModifiedCatalog {
+
+    private ShortCircuitCgmesConformity1ModifiedCatalog() {
+    }
+
+    // BaseCase is taken from powsybl-core repository
+    public static TestGridModelResources smallGridBusBranchWithBusbarSectionsAndIpMax() {
```

Comment:
You can move this in `CgmesConformity1ModifiedCatalog`. Consequently, the modified files can be moved too.